script/hassfest/quality_scale.py

```diff
@@ -752,6 +752,7 @@ class Rule:
     "octoprint",
     "oem",
     "ohmconnect",
+    "ohme",
```

Comment:
Please add a quality scale file instead. New integrations require Bronze tier:

https://developers.home-assistant.io/docs/core/integration-quality-scale/

https://developers.home-assistant.io/docs/core/integration-quality-scale/#keeping-track-of-the-implemented-rules

Reply:
Thanks for the quick response! I've got it to a stage where I believe it meets bronze tier.